src/api/backend.js

```diff
@@ -7,8 +7,10 @@ function canReadURLFromEnv () {
 }
 
 export function setBaseURLWithDefaultOrEnvValue () {
-  const defaultUrl = 'https://kubeportal-dev.api.datexis.com/'
-  return canReadURLFromEnv() ? process.env['VUE_APP_BASE_URL'] : defaultUrl
+  const API_VERSION = 'v1.0.0'
```

Comment:
I assume you change this to 1.1.0 when everything is done.

Reply:
sure